flake.nix

```diff
@@ -44,6 +46,17 @@
       npmlock2nixOverlay = final: prev: {
         npmlock2nix = import npmlock2nix-src { pkgs = prev; };
       };
+ 
+
+      pkgsForSystem = forAllSystems( system: import nixpkgs {
+           # ./overlay.nix contains the logic to package local repository
+           overlays = [ mvn2nix.overlay (
+               final: prev: {
+               tikaExtractor = final.callPackage ./tikaExtractor.nix { };
```

Comment:
could you please add this in the main overlay, right where tika-server was ?

Reply:
i have added the same.